test/description-render-test.js

```diff
@@ -2,7 +2,7 @@
 
 var assert = require('chai').assert;
 
-var markdownRenderer = require('../lib/index.js');
+var { markdownRenderer } = require('./markdownRendererWithDom.js');
```

Comment:
The test should deal with the `../lib/index.js` here.  I'd just add global.window as follows:

```suggestion
global.window = (new JSDOM('')).window;
var { markdownRenderer } = require('../lib/index.js');
```